portal/integrated-windows-authentication/README.md

```diff
@@ -1,47 +1,41 @@
-# Integrated windows authentication
+# Integrated Windows Authentication
 
-Use Windows credentials to access services hosted on a portal secured with Integrated Windows Authentication (IWA).
+Connect to an IWA secured portal and search for maps.
 
-![](IntegratedWindowsAuthentication.png)
+![Image of integrated windows authentication](IntegratedWindowsAuthentication.png)
 
 ## Use case
 
-IWA, which is built into Microsoft Internet Information Server (IIS), allows users to easily authenticate for different services using their domain's Windows credentials. This simplifies logging in to services that are provided through an intranet environment, as only a single username and password combination are required for the applications that support IWA.
+Your organization might use Integrated Windows Authentication (IWA) to secure ArcGIS Enterprise. This can be useful because the same credentials used to log into your work computer and network can be used to authenticate with ArcGIS. IWA is built into Microsoft Internet Information Server (IIS) and works well for intranet applications but isn't always practical for internet apps.
 
 ## How to use the sample
 
-Enter a URL to your IWA-secured portal into the URL field and select "Search IWA Secured Portal" to search for web maps stored there. Authenticate with a username (including domain, such as username@DOMAIN or domain\username), and password. If authentication is successful, portal item results will display in the list. Select a web map item to display it in the map view.
+Enter the URL to your IWA-secured portal. Tap the button to search for web maps stored on the portal. You will be prompted for a user name, password, and domain. If you authenticate successfully, portal item results will display in the list. Select a web map item to display it in the map view.
```

Comment:
Did we agree we should generally name the buttons, even if there is only one button?

Also a use of 'Tap' snuck in here.

```suggestion
Enter the URL to your IWA-secured portal. Click the 'Search IWA Secured Portal' button to search for web maps stored on the portal. You will be prompted for a user name, password, and domain. If you authenticate successfully, portal item results will display in the list. Select a web map item to display it in the map view.
```

Reply:
I don't think we agreed on buttons. It makes sense when there are multiple UI interactions to name the button, but when there is only one button to refer to it just as the button. Here since you can also click on a list I'm tempted to keep the name in, as you've suggested. Haha, oops, love a good mistyped tap/click!